shed/schema.go

```diff
@@ -114,7 +114,8 @@ func (db *DB) schemaIndexPrefix(name string) (id byte, err error) {
 
 // getSchema retrieves the complete schema from
 // the database.
-func (db *DB) getSchema() (s schema, err error) {
+// This is exported for displaying the schema info outside
+func (db *DB) GetSchema() (s schema, err error) {
```

Comment:
revert this please.

Reply:
Why? We need this to show the schema details in the ShowDatabaseInformation command present in localstore.go. ShowDatabaseInformation was very useful to see the entire metadata of a database.

Reply:
This function is not exported now.